samples/snippets/create_training_pipeline_image_classification_sample.py

```diff
@@ -14,8 +14,8 @@
 
 # [START aiplatform_create_training_pipeline_image_classification_sample]
 from google.cloud import aiplatform
-from google.protobuf import json_format
-from google.protobuf.struct_pb2 import Value
+from google.cloud.aiplatform.v1beta1.schema.trainingjob import definition
```

Comment:
is it possible to remove `v1beta` by exporting schema under cloud.aiplatform? (without moving all the files)

Reply:
Not very easily, no.